packages/xchain-crypto/package.json

```diff
@@ -54,6 +57,7 @@
   "dependencies": {
     "bech32": "^2.0.0",
     "bip39": "^3.0.4",
+    "bip39-converter": "^1.0.3",
```

Comment:
@MartyHav Where is the source code of `bip39-converter` published? Could not find any information at https://www.npmjs.com/package/bip39-converter nor at https://github.com/Coinomi/bip39-coinomi/

Reply:
Just found it https://github.com/haven-protocol-org/bip39mnemonicConverter

Pls add this repo to https://www.npmjs.com/package/bip39-converter

Reply:
will be added with next version